common/djangoapps/student/views/management.py

```diff
@@ -58,7 +58,7 @@
 from common.djangoapps.student.models import (  # lint-amnesty, pylint: disable=unused-import
     AccountRecovery,
     CourseEnrollment,
-    PendingEmailChange,
+    PendingEmailChange,  # unimport:skip
```

Comment:
why do we need this here

Reply:
So that unimport doesn't remove any imports from this statement